src/pages/NewChatConfirmPage.tsx

```diff
@@ -144,9 +144,9 @@ function NewChatConfirmPage({newGroupDraft, allPersonalDetails}: NewChatConfirmP
                     sections={[{title: translate('common.members'), data: sections}]}
                     ListItem={InviteMemberListItem}
                     onSelectRow={unselectOption}
-                    showConfirmButton={selectedOptions.length > 1}
+                    showConfirmButton={selectedOptions.length > 0}
```

Comment:
@abzokhattab Can we get rid of this condition?
```suggestion
                    showConfirmButton={selectedOptions.length}
```

Reply:
@abzokhattab typescript check is failing. Can you please solve the issue?

Reply:
Oh i see, the issue is now fixed